app/controllers/api/v1/scholarship_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class ScholarshipApplicationsController < ApplicationController
+      respond_to :json
+
+      def create
+        @scholarship_application = ScholarshipApplication.new(reason: params[:reason], terms_accpeted: params[:terms_accepted], scholarship_id: params[:scholarship_id], user_id: current_user.id)
```

Comment:
Instead of passing in all of the attributes here, you can leverage strong params.  For example:

```ruby
def scholarship_application_params
  params.require(:scholarship_application).permit(:reason, :terms_accepted, :scholarship_id, :user_id)
end
```

If the params do not have the attributes nested within a `scholarship_application` key (i.e. 
`{ scholarship_application: { reason: ..., user_id: ... } }`, then you can do a less strict version of the strong params like this:

```ruby
def scholarship_application_params
  params.permit(:reason, :terms_accepted, :scholarship_id, :user_id)
end
```


And then you can just call:

```ruby
ScholarshipApplication.create! scholarship_application_params
```